### ISSUE TITLE
Presence Detection based on connected clients

### ISSUE THREAD

**Issue by omarahm** [rroller/netgear#23]
**Is your feature request related to a problem? Please describe.**
I want to detect presence based on connected devices (using MAC addresses). 

**Describe the solution you'd like**
Expose one entity per MAC address to show the connected/disconnected status.

**Additional context**
I can put some effort into building this out if it is not already in the works.

**Comment by wimj888** [comment]
Would be interested as well

**Comment by Fl0kkie** [comment]
Yep, me too! device_tracker on client mac addresses would be great.

**Comment by rroller** [comment]
This is now added to the latest version. You can see connected clients in the activity feed or the "Connected Clients → Details". It's in both places.
